src/provider.ts

```diff
@@ -8,6 +8,30 @@ import { CompletionProvider } from './completion-provider';
 import { getChatModel, IBaseCompleter } from './llm-models';
 import { IAIProvider } from './token';
 
+export const chatSystemPrompt = (provider_name: string) => `
```

Comment:
Should this function take an option, instead of raw parameters like `provider_name`?

Wondering if this could make it easier to extend later.